debian/entry.sh

```diff
@@ -90,6 +90,8 @@ function init_non_systemd()
 		"$CMD" "$@" &
 		pid=$!
 		wait "$pid"
+		exit_code=$?
+		fg > /dev/null || exit "$exit_code"
```

Comment:
How does this fix the case where a user does `docker run -it ... /bin/bash`? From what I understand `/bin/bash` will be run in the background and then we'll block at `wait "$pid"`. And only after this happens will `fg` run.